draft-ietf-quic-recovery.md

```diff
@@ -134,6 +134,11 @@ Crypto Packets:
 
 : Packets containing CRYPTO data sent in Initial or Handshake
   packets.
+  
+Out-of-order Packets:
+
+: Packets that do not increase the largest received packet number for its
+cryptographic context by exactly one.
```

Comment:
Do you want to add: "Packets can arrive out of order as a result of packets ahead of them being lost or delayed."  I think that's accurate.

Reply:
I'd prefer "Packets arrive out when later packets are lost or delayed."

Reply:
I think that there are two cases we care about, I only caught the first

- an earlier packet was delayed - so there is a gap before this one
- this packet was delayed - and we already received a higher-numbered one

Maybe just: "Packets arrive out of order when packets are lost or delayed."  Which doesn't really capture all the reasons, but might suffice.

Reply:
My suggestion should have been "Packets arrive out of order when later packets are lost or delayed.", but Martin's suggestion is slightly better because it drops 'later', which is unnecessary.

The intent was as a replacement for Martin's suggestion, in addition to the first sentence you have.